logdna/package.json

```diff
@@ -3,9 +3,13 @@
   "version": "2.0.3",
   "description": "Lambda Functions to Stream Logs from AWS CloudWatch to LogDNA",
   "main": "index.js",
+  "scripts": {
+    "lint": "./node_modules/.bin/eslint -c .eslintrc *.js"
+  },
   "dependencies": {
     "agentkeepalive": "^4.0.2",
     "async": "^2.6.2",
+    "eslint": "^6.6.0",
```

Comment:
Don't add this to `package.json` - adding it to `.circleci/config.yml` is enough for the already-spoken reasons